### ISSUE TITLE
Incorrect healthcheck config in the template

### ISSUE THREAD

**Issue by Tadas** [tcomerma/ansible-keepalived#4]
I think the provided config is incorrect and will not work as intended due to incorrect priorities and check script weight:

- Master has priority of 100, BACKUP has priority of 50
- vrrp_script chk_service has weight of 2 - there is no way a failed check can make BACKUP priority > MASTER priority

Change the script weight to -51 or tune the priorities so that the script can influence which node has more priority.

https://serverfault.com/questions/718132/keepalived-vrrp-script-not-failing-over

**Comment by csayler** [comment]
> I think the provided config is incorrect and will not work as intended due to incorrect priorities and check script weight:
> 
> * Master has priority of 100, BACKUP has priority of 50
> * vrrp_script chk_service has weight of 2 - there is no way a failed check can make BACKUP priority > MASTER priority
> 
> Change the script weight to -51 or tune the priorities so that the script can influence which node has more priority.
> 
> https://serverfault.com/questions/718132/keepalived-vrrp-script-not-failing-over

Yep that is correct master weight should be 1 point higher than slaves, 51 or 101.
